src/components/NewOrderAmortization.vue

```diff
@@ -38,12 +38,12 @@
                 {{ `${customer.first_name} ${customer.last_name}` }}
               </td>
               <th>{{ order.order_number }}</th>
-              <th>{{ order.product.name }}</th>
+              <th>{{ isBNPL ? order.bnpl_vendor_product.name : order.product.name }}</th>
               <th>{{ order.owner }}</th>
               <th>{{ order.serial_number }}</th>
               <th>{{ order.business_type }}</th>
               <th class="text-capitalize">
-                {{ order.down_payment_rate }} percent
+                {{ ((order.down_payment / order.product_price) * 100).toFixed(0) }} percent
```

Comment:
Why are we doing this calculation here?

Reply:
the bnpl orders don't have the downpayment attached to it. And rather than create another component for it. I had to come up with that